core/vm/opcodes.go

```diff
@@ -204,6 +207,12 @@ const (
 	LOG4
 )
 
+// 0xb0 range - control flow ops.
+const (
+	CALLF = 0xb0 + iota
+	RETF
+)
+
```

Comment:
`TLOAD` and `TSTORE` are in the `0xb0` range as well and are defined below the `0xf0` range, probably a good idea to consolidate them into a single group